packages/form-upload/Upload.js

```diff
@@ -308,6 +310,7 @@ Upload.propTypes = {
   customerId: PropTypes.string.isRequired,
   deliverFileOnSubmit: PropTypes.bool,
   deliveryChannel: PropTypes.string,
+  onFilePreUpload: PropTypes.func,
```

Comment:
This should be an array of functions since it is being passed to `onPreStart` in `upload-core` https://github.com/Availity/sdk-js/blob/9b5e81bef0216d13fab6ca0bbe87370e2019f490/docusaurus/docs/api/uploads.md#upload-object-variables-that-can-be-set-before-call-to-start